staging/src/k8s.io/legacy-cloud-providers/gce/gce_instances.go

```diff
@@ -561,7 +561,7 @@ func (g *Cloud) getInstancesByNames(names []string) ([]*gceInstance, error) {
 		return nil, err
 	}
 	if len(foundInstances) != len(names) {
-		return nil, cloudprovider.InstanceNotFound
+		klog.Warningf("getFoundInstanceByNames - input instances %d, found %d. Continuing LoadBalancer Update", len(names), len(foundInstances))
```

Comment:
Maybe not an issue - would there be any risk that we failed to find any node mistakenly and still update the LB?

Reply:
The one risk I can think of - If GCE instance get calls are failing due to some reason (rate limiting or backend is really down), we will end up updating the TargetPool with 0 instances. Made a change to return error if we got 0 instances back.